src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java

```diff
@@ -579,36 +582,49 @@ static SearchSourceBuilder byggStatusTallForVeilederQuery(String enhetId, String
         return new SearchSourceBuilder()
                 .size(0)
-                .aggregation(
-                        filters(
-                                "statustall",
-                                erSykemeldtMedArbeidsgiverFilter(filtrereVeilederOgEnhet, vedtakstottePilotErPa),
-                                mustExistFilter(filtrereVeilederOgEnhet, "iavtaltAktivitet", "aktiviteter"),
-                                mustExistFilter(filtrereVeilederOgEnhet, "iAktivitet", "alleAktiviteter"),
-                                ikkeIavtaltAktivitet(filtrereVeilederOgEnhet),
-                                inaktiveBrukere(filtrereVeilederOgEnhet),
-                                mustBeTrueFilter(filtrereVeilederOgEnhet, "minArbeidsliste", "arbeidsliste_aktiv"),
-                                mustBeTrueFilter(filtrereVeilederOgEnhet, "nyeBrukereForVeileder", "ny_for_veileder"),
-                                totalt(filtrereVeilederOgEnhet),
-                                trengerVurderingFilter(filtrereVeilederOgEnhet, vedtakstottePilotErPa),
-                                mustExistFilter(filtrereVeilederOgEnhet, "venterPaSvarFraNAV", "venterpasvarfranav"),
-                                mustExistFilter(filtrereVeilederOgEnhet, "venterPaSvarFraBruker", "venterpasvarfrabruker"),
-                                ufordelteBrukere(filtrereVeilederOgEnhet, veiledereMedTilgangTilEnhet),
-                                mustExistFilter(filtrereVeilederOgEnhet, "utlopteAktiviteter", "nyesteutlopteaktivitet"),
-                                moterMedNavIdag(filtrereVeilederOgEnhet),
-                                alleMoterMedNavIdag(filtrereVeilederOgEnhet),
-                                mustExistFilter(filtrereVeilederOgEnhet, "underVurdering", "utkast_14a_status"),
-                                mustMatchQuery(filtrereVeilederOgEnhet, "minArbeidslisteBla", "arbeidsliste_kategori", Arbeidsliste.Kategori.BLA.name()),
-                                mustMatchQuery(filtrereVeilederOgEnhet, "minArbeidslisteLilla", "arbeidsliste_kategori", Arbeidsliste.Kategori.LILLA.name()),
-                                mustMatchQuery(filtrereVeilederOgEnhet, "minArbeidslisteGronn", "arbeidsliste_kategori", Arbeidsliste.Kategori.GRONN.name()),
-                                mustMatchQuery(filtrereVeilederOgEnhet, "minArbeidslisteGul", "arbeidsliste_kategori", Arbeidsliste.Kategori.GUL.name())
-                        ));
+                .aggregation(filters("statustall", statusTallFiltre));
+    }
+
+    @NotNull
+    private static FiltersAggregator.KeyedFilter[] getEnhetStatusTallFiltre(BoolQueryBuilder filtrereVeilederOgEnhet, List<String> veiledereMedTilgangTilEnhet, boolean vedtakstottePilotErPa) {
+        return ArrayUtils.addAll(
+                getVeilederStatusTallFiltre(filtrereVeilederOgEnhet, veiledereMedTilgangTilEnhet, vedtakstottePilotErPa),
+                adressebeskyttelseEllerSkjermingTotalt(filtrereVeilederOgEnhet),
```

Comment:
I need explanation how this will be shown to veileders in frontend. It seems like that we are considering protected users only in combination with few filters (`Utfordelte`, `VenterPaSvarFraNav`), so I am wondering how we choosed those specific filters and not all for example?